plugins/node/opentelemetry-instrumentation-pg/src/utils.ts

```diff
@@ -44,7 +44,11 @@ function arrayStringifyHelper(arr: Array<unknown>): string {
 }
 
 // Helper function to get a low cardinality command name from the full text query
-function getCommandFromText(text?: string): string {
+function getCommandFromText(
+  config: Pick<PgInstrumentationConfig, 'getCommandFromText'>,
+  text?: string
+): string {
+  if (config.getCommandFromText) return config.getCommandFromText(text);
```

Comment:
I think this function should not depend on config, it has name `getCommandFromText` so it extracts command from text and nothing more. I think this check / call for this function from config - this should be done in `upper level` or create a new helper for that if it will be repeated in more places. WDYT ?

Reply:
I went this way around as it's quite a bit of code to inline, especially if you think `(config.getCommandFromText ?? getCommandFromText)(text)` is ugly; you'd end up adding a function for `function getCommandFromTextMaybeFromConfig(config, text) { if (config.getCommandFromText) return config.getCommandFromText(text); return getCommandFromText(text);` which seemed excessive?

Reply:
Alternatively you could override the default value in the config at construct time, but nothing else seemed to be doing that.